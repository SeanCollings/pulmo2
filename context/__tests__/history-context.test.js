import React, { useContext } from 'react';
import { create } from 'react-test-renderer';
import { cleanup } from 'react-native-testing-library';

import HistoryContextProvider, { HistoryContext } from '../history-context';

const MockComponent = () => {
  const context = useContext(HistoryContext);
  expect(context).toMatchSnapshot();
  return null;
};

describe('history-context', () => {
  afterEach(cleanup);

  test('unit test', () => {
    let context = create(
      <HistoryContextProvider>
        <MockComponent />
      </HistoryContextProvider>
    );

    expect(context).toMatchSnapshot();
  });
});
