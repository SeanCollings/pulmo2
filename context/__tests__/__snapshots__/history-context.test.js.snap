// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`history-context unit test 1`] = `
Object {
  "addActivity": [Function],
  "deleteActivity": [Function],
  "favouriteActivity": [Function],
  "getActivitiesByMonth": [Function],
  "getActivityByDate": [Function],
  "getFavActiviesBySlice": [Function],
  "getSavedActivitiesBySlice": [Function],
  "updateActivity": [Function],
  "updatedActivity": null,
}
`;

exports[`history-context unit test 2`] = `null`;
